source/config/configSpec.py

```diff
@@ -47,6 +47,7 @@
 	autoDialectSwitching = boolean(default=false)
 
 	[[__many__]]
+		extraDescriptions = integer(default=0, min=0, max=5000)

```

Comment:
```suggestion
		autoCharacterDescriptions = integer(default=0, min=0, max=5000)

```

Reply:
> ## Some personal opinions:
> 1. Instead of directly entering mil values, you should use a combo box with quarter second increments from "no delay" to five seconds. Playing with ms values seems like too high a precision for users to have to worry about.

I agree! Thanks, that's not a bad idea, I just need to loop through the values from .25 to 5000. Maybe I can even use range() for that :-) I write typescript in my day job now, so python is a nice blast to the past.

> 2. The selector name, "Extra description pause time (ms)", doesn't feel right. Perhaps something like "Character description delay", or "Automatic character description delay"?


I accepted your suggestions.

I also saw your comment on it not working properly in espeak. Please see my pr description.